### ISSUE TITLE
Support Eclipse "References" command to find/show all usage of a definition

### ISSUE THREAD

**Issue by tedepstein** [RepreZen/KaiZen-OpenAPI-Editor#219]
Request from Saul Margolis:

A useful Eclipse feature for Java and other languages.  It would be helpful to support this in SwagEdit.  This is Ctrl+Shift+G "Find References" in Window/JDT.  Also works in RAPID-ML and other XText-based editors.  

I think the implementation of this feature also provides some of the work required to support Refactoring (see #220), another useful capability.

**Comment by tfesenko** [comment]
I completely agree that "find references" is very useful. In Eclipse it has severals levels: 
<img width="479" alt="screen shot 2016-10-06 at 1 14 05 pm" src="https://cloud.githubusercontent.com/assets/644582/19162668/cb06ded8-8bc6-11e6-93f3-6b1dec4baa1b.png">

Do we want to support all of them with the same look-n-feel? Or, will "Find Reference in Workspace" be sufficient?

**Comment by tedepstein** [comment]
We can decide and prioritize later, this is not in scope for a current sprint.
